### PR TITLE
[newrelic-logging]: Allow extraVolumes/extraVolumeMounts configuration options

### PR DESCRIPTION
**Pull request by axilleastr**
#### Is this a new chart
No.

#### What this PR does / why we need it:
Adds two configuration options for the `[newrelic-logging]` chart:
* extraVolumeMounts
* extraVolumes

#### Which issue this PR fixes
  - fixes https://github.com/newrelic/helm-charts/issues/956

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Comment by michaelsilver** [comment]
Friendly bump on this PR :)

File: charts/newrelic-logging/values.yaml [hunk 1]
```diff
@@ -239,3 +239,12 @@ daemonSet:
...
 # lowDataMode: false
+
+extraVolumes: []
+# - name: systemdlog
+#   hostPath:
+#     path: /run/log/journal
+
+extraVolumeMounts: []
+# - name: systemdlog
+#   mountPath: /run/log/journal
```
Comment (jsubirat) [inline charts/newrelic-logging/values.yaml:250]: Please also add a tree for Windows:
```
windows:
    extraVolumes: []
    extraVolumeMounts: []
```

In the future, we will move all the windows-related values here (such as the `windowsNodeSelector`, `windowsOsList`, `enableWindows`...).

**Review by jsubirat — CHANGES_REQUESTED** [review]
Can you please address these little comments? Thanks!

File: charts/newrelic-logging/templates/daemonset.yaml [hunk 2]
```diff
@@ -112,6 +112,9 @@ spec:
               mountPath: /fluent-bit/etc
             - name: var
               mountPath: /var
+          {{- if .Values.extraVolumeMounts }}
```
[the remark this answers is not in the document]
Comment (bmcfeely) [reply charts/newrelic-logging/templates/daemonset.yaml:115]: @jsubirat would it be harmful to merge this, and then create another PR for windows? (just so the customer can get what they need sooner)

[the remark this answers is not in the document]
Comment (jsubirat) [reply charts/newrelic-logging/templates/daemonset.yaml:115]: This also works for  me, let me get this merged then

**Review by jsubirat — APPROVED** [review]

File: charts/newrelic-logging/templates/daemonset.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (axilleastr) [reply charts/newrelic-logging/templates/daemonset.yaml:115]: hm, @jsubirat, it seems that release failed due to a chart version conflict i guess (?) 
https://github.com/newrelic/helm-charts/actions/runs/3462547940/jobs/5781589624
Should we bump up newrelic-logging chart's version?

**Pull request merged**